client/dashboard/sites/backups/actions.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon } from '@wordpress/components';
```

Comment:
So there's this recurring pattern where we have separate file(s) for dataviews-related logic. But currently we're not consistent.

So I suggest we allow the following options:

- put the dataviews logic in the component itself if it's small enough,
- or put in one file `dataviews.tsx`,
- or split into multiple files in a `dataviews/` directory (like `dataviews/{fields,actions}.tsx`).

This is because e.g. `actions` alone is not descriptive enough.

So for this PR, I suggest to put `actions.tsx` and `fields.tsx` inside `client/dashboard/sites/backups/dataviews/`.

And for consistency, @Automattic/lego we need to fix the dataviews-related files for `sites`. For `domains` and `purchases`, it's already correct .